landingzones/tests/test_permissions.py

```diff
@@ -200,6 +200,83 @@ def test_zone_create_disable(self):
         self.assert_response(url, good_users, 200)
         self.assert_response(url, bad_users, 302)
 
+    def test_zone_update(self):
+        """Test ZoneUpdateView permissions"""
+        url = reverse(
+            'landingzones:update',
+            kwargs={'landingzone': self.landing_zone.sodar_uuid},
+        )
+        good_users = [
+            self.superuser,
+            self.user_owner_cat,
+            self.user_owner,
+            self.user_delegate,
+            self.user_contributor,
+        ]
+        bad_users = [
+            self.user_guest,
+            self.anonymous,
```

Comment:
Minor point: we should put `user_no_roles` before `anonymous` for the sake of logic. These have previously been the wrong way around in some cases, but I've been trying to fix this whereever I encounter it.